### PR TITLE
Chart Remake - ICU Utilization

### PR DESCRIPTION
**Pull request by pnavarrc**
Please review this after #790

NOTE: #790 needs to be merged first, and then we need to change the base branch to `develop` in this PR

### CODE REVIEW

File: src/components/Charts/ChartZones.tsx [hunk 1]
```diff
@@ -184,12 +186,14 @@ const ChartZoneAutosize = ({
   zones,
   capY,
   getTooltipText,
+  getPointText = (valueY: number) => formatPercent(valueY, 1),
```
Comment (goldblatt) [inline src/components/Charts/ChartZones.tsx:189]: why do you set the default here?

Comment (goldblatt) [reply src/components/Charts/ChartZones.tsx:189]: maybe we can add a comment i think this sets the floats to show

File: src/components/Charts/ChartICUHeadroom.tsx [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+const getTooltipText = (valueY: number) =>
+  valueY > CAP_Y
+    ? `ICU headroom used > ${getPointText(CAP_Y)}`
+    : `ICU headroom used ${getPointText(valueY)}`;
+
+const ChartICUHeadroom = ({
+  columnData,
+  height = 400,
+}: {
+  columnData: any[];
```
Comment (goldblatt) [inline src/components/Charts/ChartICUHeadroom.tsx:19]: can we give this a type

File: src/components/Charts/ChartZones.tsx (hunk 1 above)
Comment (pnavarrc) [reply src/components/Charts/ChartZones.tsx:189]: is not really that useful, removing it as default and moving it to its corresponding chart

**Comment by pnavarrc** [comment]
Thanks again @goldblatt - updated with your suggestions, merging!

**Pull request merged**